cachi2/core/package_managers/yarn/main.py

```diff
@@ -35,13 +34,16 @@ def _resolve_yarn_project(project: Project, output_dir: RootedPath) -> list[Comp
     _configure_yarn_version(project)
 
     try:
-        _set_yarnrc_configuration(project, output_dir)
-        packages = resolve_dependency_info(project.source_dir)
+        _disable_plugins(project.yarn_rc)
+
+        cli = YarnCli(project.source_dir, _generate_yarn_env(project, output_dir))
+
+        packages = resolve_dependency_info(cli)
```

Comment:
This just looks weird to me (i.e. passing in a CLI object)...

Reply:
So, in other words, I agree with Taylor and Adam that using the rc file makes more sense :grin: